apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -398,14 +401,7 @@ export class TLDrawDurableObject extends DurableObject {
 						openMode = ROOM_OPEN_MODE.READ_ONLY
 					}
 				}
-			} else if (!this.documentInfo.appMode) {
```

Comment:
Feels like we don't need this any longer? If we didn't go through the proper file creation flow then the file won't exists in the db and then loading the file record will fail. Tried it by logging out then going to `/f/random-id` and I was correctly shown the `Not found` page.